UI/item-widget-helpers.cpp

```diff
@@ -37,14 +37,18 @@ void DeleteListItem(QListWidget *widget, QListWidgetItem *item)
 
 void ClearListItems(QListWidget *widget)
 {
+#if QT_VERSION < QT_VERSION_CHECK(6, 4, 3)
 	// Workaround for the SceneTree workaround for QTBUG-105870
 	widget->setProperty("clearing", true);
+#endif
 
 	widget->setCurrentItem(nullptr, QItemSelectionModel::Clear);
 
 	for (int i = 0; i < widget->count(); i++)
 		delete widget->itemWidget(widget->item(i));
 
 	widget->clear();
+#if QT_VERSION < QT_VERSION_CHECK(6, 4, 3)
 	widget->setProperty("clearing", false);
```

Comment:
Maybe add a comment here that this is also related to QTBUG-105870 just to make it clear.

Reply:
Added.